x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/response_actions_log.test.tsx

```diff
@@ -201,6 +201,73 @@ describe('Response Actions Log', () => {
       ).toEqual(['Time', 'Command', 'User', 'Hosts', 'Comments', 'Status', 'Expand rows']);
     });
 
+    it('should show multiple hostnames correctly', async () => {
+      const data = await getActionListMock({ actionCount: 1 });
+      data.data[0] = {
+        ...data.data[0],
+        hosts: {
+          ...data.data[0].hosts,
+          'agent-b': { name: 'Host-agent-b' },
+          'agent-c': { name: '' },
+          'agent-d': { name: 'Host-agent-d' },
+        },
+      };
+
+      mockUseGetEndpointActionList = {
+        ...baseMockedActionList,
+        data,
+      };
+      render({ showHostNames: true });
+
+      expect(renderResult.getByTestId(`${testPrefix}-column-hostname`)).toHaveTextContent(
```

Comment:
Can you add a test case like this one, but set `Host-agent-b` to not have a name, so that you are validating that the `Host unenrolled` is being used.

Reply:
https://github.com/elastic/kibana/pull/139797/commits/e05903d8011a31dbdf1448461ef6a1f3204c8f26#r960292157